src/components/ComponentPages/TopicDetails/index.tsx

```diff
@@ -37,19 +39,19 @@ const TopicDetails = () => {
           algorithm: algorithm,
           update_all: 0,
         };
-        const result = await getTreesApi(reqBody);
+        await getTreesApi(reqBody);
       } else didMount.current = true;
     }
     getTreeApiCall();
   }, [asofdate, algorithm]);
 
-  const reqBody = {};
+  const reqBody = { topic_num: 45, camp_num: 1 };
```

Comment:
Is it just a mock data?

Reply:
Yeah these are the mock params and will be replaced with actual one later